services/wallet/router/pathprocessor/processor_swap_paraswap.go

```diff
@@ -22,9 +22,8 @@ import (
 type SwapParaswapTxArgs struct {
 	transactions.SendTxArgs
 	ChainID            uint64  `json:"chainId"`
-	ChainIDTo          uint64  `json:"chainIdFrom"`
-	TokenIDFrom        string  `json:"tokenIdFrom"`
-	TokenIDTo          string  `json:"tokenIdTo"`
+	FromSymbol         string  `json:"fromSymbol"`
```

Comment:
@dlipicar I agree with `Symbol` from/to for parameter name, but `TokenID` to/from was used to align with other types.

Reply:
hmm I'd avoid using the term "Token ID" for anything other than the ID used to specify a collectible from an ERC721/1155 contract, that's a term defined in the standards...
Also, this struct is Swap- (even Paraswap)-specific, not sure why we would care so much about alignment with other parts.

Reply:
Well, that's true, but since we're using the same `CreateMultiTransaction` function for sending different sending types just thinking that we should be aligned.

But as we discussed once, I hope that in 2.31 will be time to make a uniform type that will support all of them, this way is a bit chaotic, and this part will be something we plan to work on very soon (in a couple of weeks). Referring to improving this type (basically all of them contain more or less the same/similar set of data, no need to have separate types for all of them, and each type has `transactions.SendTxArgs` as parent type, we will optimize that):
```
type MultipathProcessorTxArgs struct {
	Name              string `json:"bridgeName"`
	ChainID           uint64
	TransferTx        *transactions.SendTxArgs
	HopTx             *HopBridgeTxArgs
	CbridgeTx         *CelerBridgeTxArgs
	ERC721TransferTx  *ERC721TxArgs
	ERC1155TransferTx *ERC1155TxArgs
	SwapTx            *SwapParaswapTxArgs
}
```

Reply:
Alright! I'll just trust your judgement here, we can discuss it further when these types get reworked. I'll revert to the original struct